core-bundle/src/Resources/contao/languages/pl/tl_article.xlf

```diff
@@ -111,31 +111,31 @@
       </trans-unit>
       <trans-unit id="tl_article.cssID.1">
         <source>Here you can set an ID and one or more classes.</source>
-        <target>Jeśli potrzebujesz osobnej definicji do formatowania tego modułu w stylach CSS, możesz tu wprowadzić ID. To ID będzie przypisane do tego modułu i można je modyfikować w arkuszu CSS.</target>
+        <target>Jeśli potrzebujesz osobnej definicji do formatowania tego modułu w stylach CSS, możesz tu wprowadzić ID. To ID będzie przypisane do tego modułu i można je opisywać w arkuszu CSS.</target>
```

Comment:
Maybe `używać` would be more appropriate after all.

Reply:
`używać` is fine